packages/reporter/src/runnables/runnable-header.tsx

```diff
@@ -46,9 +51,17 @@ const RunnableHeader: React.FC<RunnableHeaderProps> = observer(({ spec, statsSto
     relativeFile: relativeSpecPath,
   }
 
+  const openInIDE = () => {
```

Comment:
nit: should use the naming convention for components here and render it using JSX notation

```suggestion
  const OpenInIDE = () => {
```
...

```
<OpenInIDE />
```